news/2018-11-04-project-loved-week-of-november-4th.md

```diff
@@ -0,0 +1,208 @@
+
+In case you don't read the end of this post, be sure to submit suggestions for beatmaps to be added to Loved using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! All of the nominated beatmaps are chosen from submissions on this form, which you can view in the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit#gid=0).
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050), [DigitalHypno](https://osu.ppy.sh/users/4384207), [SoraRoxas01](https://osu.ppy.sh/users/1986262) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/sakura-trip.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Aika - Sakura Trip](https://osu.ppy.sh/beatmapsets/830964#osu) by [CrazyrabbitKGE](https://osu.ppy.sh/users/4481526)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+CrazyrabbitKGE is a name unfamiliar to most, but I promise you it's a name you'll want to remember. His map of "Aika - Sakura Trip" provides a challenging twist for experienced players and sets the canvas for a unique spectator experience with its Sakura flower pattern structure and various keysounded sections. The map garnered over 100 favorites just through word of mouth, so if you want to know what the buzz is about, check out the map and vote for its place in the Loved category!
+
+[![](/wiki/shared/news/2018-11-04-project-loved-week-of-november-4th/osu/speedom-wwc2-feat-krizz-kaliko-and-eminem.jpg)](https://osu.ppy.sh/community/forums/topics/TOPIC_ID)
+
+[Tech N9ne - Speedom (WWC2) (feat. Krizz Kaliko and Eminem)](https://osu.ppy.sh/beatmapsets/397474#osu) by [LukerMaster](https://osu.ppy.sh/users/2093623)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Some people may recognize this mapper from another Loved map, "Imaginary Places", which is iconic for its Boss level difficulty. This map is no different. Clocking in over 8 stars, this alt-styled fast aim map by LukerMaster will push you to your limits. Good luck!
```

Comment:
```suggestion
Some people may recognize this mapper from another Loved map, "Imaginary Places", which is iconic for its boss level difficulty. This map is no different. Clocking in over 8 stars, this alt-styled fast aim map by LukerMaster will push you to your limits. Good luck!
```

---
I don't think there is a need to first letter capitalise "boss"

Reply:
ah I thought it was a diff name, I guess not